### PR TITLE
[image_picker]fix touches event through UIImagePickerController down to Flutter Widgets

### PR DESCRIPTION
**Pull request by zzzs123**
## Description

In a Flutter widget and present image_picker to take a photo, after shot and touch the photo, the touch event will be passed through UIImagePickerController. If there's a TextField in flutter under the UIImagePickerController, the keyboard will be presented.

## Related Issues
https://github.com/flutter/flutter/issues/32896
https://github.com/flutter/flutter/issues/54251

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by zzzs123** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fflutter%2Fplugins%2Fpull%2F2640) for more info**.

@googlebot I signed it!

**Comment by zzzs123** [comment]
@googlebot I fixed it.

**Comment by renefloor** [comment]
Looks like a clean (temporary) fix, but maybe somebody with more iOS experience should verify this.
Note that also somebody else is working on this to fix for Flutter in general as this is a problem for all viewcontrollers: https://github.com/flutter/flutter/issues/14720#issuecomment-862709250

File: packages/image_picker/ios/Classes/FLTImagePickerController.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//
+//  FLTImagePickerController.h
+//  image_picker
+//
+//  Created by Zhang Yanjun, (Yanjun.Zhang@partner.bmw.com) on 2020/4/8.
+//
```
Comment (ydag) [inline packages/image_picker/ios/Classes/FLTImagePickerController.h:6]: You can remove this and add :

>  Copyright 2013 The Flutter Authors. All rights reserved.
>  Use of this source code is governed by a BSD-style license that can be
>  found in the LICENSE file.

File: packages/image_picker/ios/Classes/FLTImagePickerController.m [hunk 2]
```diff
@@ -0,0 +1,35 @@
+//
+//  FLTImagePickerController.m
+//  image_picker
+//
+//  Created by Zhang Yanjun, (Yanjun.Zhang@partner.bmw.com) on 2020/4/8.
+//
```
Comment (ydag) [inline packages/image_picker/ios/Classes/FLTImagePickerController.m:6]: same here.

**Review by ydag — COMMENTED** [review]
Thanks for the PR. We are discussing this issue [here](https://github.com/flutter/flutter/issues/14720) if you want to check. I left some comments :)

**Comment by stuartmorgan** [comment]
Thank you for your contribution. I'm going to close this PR for now since there are outstanding comments, just to get this off our PR review queue. Please don't hesitate to submit a new PR if you have the time to address the review comments. Thanks!

**Pull request closed without merging**